items.md

```diff
@@ -123,6 +122,20 @@ Administrators can directly create an archived item (bypassing the workflow). An
 }
 ```
 
+### Creating an archived item from an external source
+**POST /api/core/items?owningCollection=<:uuid>**
+
+Administrators can directly create an archived item (bypassing the workflow) from an external source. The content-type is uri-list.
+
+The URI-list should contain the [external entry value](external-authority-sources.md) whose metadata should be imported
```

Comment:
One minor thing we should note.  The uri-list should only contain ONE entry.  This endpoint should throw an error (likely a 400 Bad Request) if multiple entries are passed in the uri-list.

Reply:
I've included the note